_posts/2019-04-29-opt-cuda-quantized.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "Automating Optimization of Quantized Deep Learning Models on CUDA"
+date:   2019-04-29 00:00:00 +0800
+author: Wuwei Lin
+---
+Deep learning has been successfully applied to a variety of tasks.
+On real-time scenarios such as inference on autonomous vehicles, the inference speed of the model is critical.
+Network quantization is an effective approach to accelerating deep learning models.
+In quantized models, both data and model parameters are represented with low precision data types such as `int8` and `float16`.
+The lowered data bandwidth reduces the inference time and memory/storage requirements, as well as the power consumption.
+Meanwhile, under proper quantization schemes, we can minimize the accuracy drops of the quantized models.
+Therefore, quantized operators are of particular interests of researchers and developers as it makes large models suitable to deploy on diverse devices, such as GPU, CPU and mobile devices.
+
+Previously, quantized operators are usually optimized with handcrafted microkernels for different workloads.
```

Comment:
workloads, or relies on blackbox proprietary solutions such as cuDNN and TensorRT.